### PR TITLE
[SD-2046] Allow a stage to be marked as a published stage one per desk

### PR DESCRIPTION
**Pull request by marwoodandrew**
This change allows you to nominate a (only one) stage in a desk as a published stage. The intention is that when an item is published it will be moved to this stage.

I thought it would be nice to be able to change which stage is the default incoming stage as well!

### CODE REVIEW

File: client/app/scripts/superdesk-desks/views/desk-config-modal.html [hunk 1]
```diff
@@ -111,6 +111,12 @@ <h3 ng-show="desk.edit._id" translate>Edit "{{desk.edit.name}}" Desk</h3>
 							<label translate>Global Read: <label class="label globalread">
 								{{(selected.is_visible == null || selected.is_visible) ? 'ON' : 'OFF'}}
 							</label></label>
+							<label class="label globalread" ng-if="selected.default_incoming == null || selected.default_incoming">
+								{{:: 'Default incomming stage' }}
+							</label>
+							<label class="label globalread" ng-if="selected.published_stage !== null && selected.published_stage">
```
Comment (plamut) [inline client/app/scripts/superdesk-desks/views/desk-config-modal.html]: The first part of the `ng-if` expression is redundant?

**Comment by ioanpocol** [comment]
:+1:

**Comment by amagdas** [comment]
:+1:

**Comment by plamut** [comment]
Hey, wait, there was a comment pending. Oh well... :)

**Comment by amagdas** [comment]
Andrew is on vacation. Will take care of the comment in another PR

**Comment by plamut** [comment]
Oh, OK, good then. It's a minor readability optimization anyway, the code itself works just fine as it is.

**Pull request merged**